python/dgl/graph_index.py

```diff
@@ -1160,16 +1157,16 @@ def from_networkx(nx_graph, readonly):
             # to_directed creates a deep copy of the networkx graph even if
             # the original graph is already directed and we do not want to do it.
             nx_graph = nx_graph.to_directed()
-    num_nodes = nx_graph.number_of_nodes()
```

Comment:
I don't think NetworkX DiGraph objects have an `num_nodes` API. Could you double check?

Reply:
Good catch, I reverted many of this mis-change, but this is missed.
Likely due to lack of test coverage.

Reply:
maybe not necessarily in this PR, perhaps we want to examine the functionality of these APIs and see if we should add unit tests to cover them